lisa/wlgen/rta.py

```diff
@@ -717,10 +728,15 @@ def __init__(self, start_pct=0, end_pct=100, delta_pct=10, time_s=1,
         phases = []
         for load in steps:
             if load == 0:
-                phase = Phase(time_s, 0, 0, cpus)
+                phase = Phase(time_s, 0, 0, cpus, uclamp_min=uclamp_min,
+                              uclamp_max=uclamp_max)
             else:
-                phase = Phase(time_s, period_ms, load, cpus)
+                phase = Phase(time_s, period_ms, load, cpus, 
+                              uclamp_min=uclamp_min, uclamp_max=uclamp_max)
             phases.append(phase)
+            # It is only necessary to set clamp values on the first phase.
```

Comment:
We don't do this for the `cpus` parameter, which AFAICT have a similar behaviour (CPU affinity carries over phases if not specified otherwise). This should be an optimization on rt-app's side I think, so maybe we can just repeatedly specify uclamp values on our side.

Reply:
OK, this is no longer done.